docs/guides/06-aggregations.md

```diff
@@ -0,0 +1,408 @@
+## Aggregating data
+
+In this guide you will explore a set of functions called **aggregations** (aggr.), that will allow you to extract different values from your datasets considering the whole set of features (*global aggr.*), just the ones in the current map extent (*viewport aggr.*) or from spatial aggregations (*cluster aggr.*).
+
+
+### Global aggregations
+When you are working with a dataset, odds are that you are interested in comparing a feature with the rest of them. For example, _"is the population density in this neighborhood bigger than the average?"_. Or maybe you just want to display in a table some basic statistics describing the whole dataset.
+
+For these cases, CARTO VL provides you with a set of functions automatically calculated for the whole dataset (hence *global* functions):
+- `globalMin`: calculates the minimum value.
+- `globalMax`: calculates the maximum value.
+- `globalSum`: calculates the sum of all values.
+- `globalCount`: calculates the number of features.
+- `globalAvg`: calculates the average value.
+- `globalPercentile`: calculates the Nth percentile.
+
+Let's use some of them with our cities dataset. You can start as usual from a [basemap](/developers/carto-vl/examples/maps/guides/getting-started/step-1.html). Take its source code and copy it to a new file called `aggregations.html`. Now you're ready to start.
+
+Then create a visualization adding the cities layer and 3 variables with global functions using this code:
+```js
+const citiesSource = new carto.source.Dataset('populated_places');
+const citiesViz = new carto.Viz(`
+    color: grey
+    width: 10
+    @g_max: globalMax($pop_max)
+    @g_avg: globalAvg($pop_max)
+    @g_p95: globalPercentile($pop_max, 95)
+`);
+const citiesLayer = new carto.Layer('cities', citiesSource, citiesViz);
+citiesLayer.addTo(map);
+```
+> The use of `g_` prefix is not required but it can be useful as an indicator of a _global_ variable. Don't forget to add your credentials as usual, using `carto.setDefaultAuth({ username: 'cartovl', apiKey: 'default_public'});` to be able to connect to the CARTO dataset.
```

Comment:
I would skip the "Don't forget to add your credentials as usual, using `carto.setDefaultAuth({ username: 'cartovl', apiKey: 'default_public'});` to be able to connect to the CARTO dataset." since it may add some noise IMO".